apps/hashdotdev/src/_pages/blog/0014_structured-data-value.mdx

```diff
@@ -0,0 +1,109 @@
+
+HASH _began life_ building agent-based modeling software. We encountered two main problems in convincing folks to adopt predictive, "generative" AI and simulations in their businesses... _back in 2019._
+
+First, the shape of folks' existing data. Of all the folks we engaged with, generally speaking _none_ had data easily available in nice, neat agent-mappable form. The closest we encountered were enteprises who'd deployed software like Palantir's and developed ["dynamic ontologies"](https://hash.ai/glossary/schemas), but even these rarely represented their entire organization's set of information in a semantic way. In the last year or so, the term "semantic layer" has become vogue term in some corners of Silicon Valley to refer to the ideal of a business whose data is enitrely neatly mapped to real-world "entities", but it remains an ambition for many, as opposed to a realized state of affairs. **We now believe foundation models will help organizations quickly 'type' their data and convert what they have to 'entities' suitable for use in simulations.**
+
+Second, simulations have historically required handoff between:
+
+1. data engineers and analysts who make the data available which is required to instantiate 'agents' in a simulation, and hydrate them with 'properties' that describe them accurately
+2. programmers who're able to encode subject matter expertise _in code_ that provides 'behavioral logic' to the agents, and environment within which they reside
+3. domain experts, who understand how the target system functions, and can actually provide the subject matter expertise and model design required.
+
+These different stakeholders and the communication/coordination/iteration loops involved in getting simulation models accurate _and useful_ were frankly killer. With data structuring and availability problems potentially already solved by foundation models (as outlined above), **we believe foundation models' further potential lies in enabling domain experts to bypass programmers and iterate on simulation logic directly.** Behaviors within simulations are generally relatively simple, standalone things. Natural language behavior developement seems eminently achievable using current-generation (e.g. GPT-4) models.
+
+The role of structured data in all this is simple.
+
+**Predictive simulations of real-world things are extremely easy to get wrong.** Many interesting systems that users typically want to simulate end up being "complex systems" in some respect, demonstrating some degree of nonlinearity due to emergent or adaptive behaviors.
+
+Complex systems are typically extremely sensitive to their initial conditions. Ensuring an as close to accurate starting or "initial state" is therefore key. Pre-structured data, especially if itself tested and reviewed, or produced in accordance with a well-understood process or pipeline, can improve confidence in this.
```

Comment:
Arguably 'data' could be plural here, but I think 'itself' can just be removed altogether to avoid the question
```suggestion
Complex systems are typically extremely sensitive to their initial conditions. Ensuring an as close to accurate starting or "initial state" is therefore key. Pre-structured data, especially if tested and reviewed, or produced in accordance with a well-understood process or pipeline, can improve confidence in this.
```